src/lib/types/openapiDoc.ts

```diff
@@ -6469,36 +6593,6 @@ const document = {
                             nullable: true,
                             description: 'A non-semver display name for the version.',
                           },
-                          git: {
```

Comment:
why was this removed?

Reply:
@julshotal Refactored it away as part of the branching project

Reply:
we don't use it on the front end & we figured when it comes to merging functionality later saving the `latestCommitDate` here wasn't something we felt confident about always being the most updated data